vue-frontend/src/components/Authentication.vue

```diff
@@ -52,28 +52,46 @@
-                      type="password" required :rules="[() => rejected ? '' : true]"></v-text-field>
-                  </v-card-text>
-                </v-card>
+                <form @submit.prevent="authenticateEducators">
+                  <v-card flat>
+                    <v-card-text>
+                      Manage your students' progress with
+                      <strong>Can I Graduate?</strong> with ease!
+                      <v-text-field name="username" label="Email address" v-model.lazy="educators.username"
+                        required :rules="[() => rejected ? '' : true]"></v-text-field>
+                      <v-text-field name="password" label="Password" v-model.lazy="educators.password"
+                        type="password" required :rules="[() => rejected ? '' : true]"></v-text-field>
+                      <p class="red--text auth-error">
+                        {{ educatorSignInMessage }}
+                      </p>
+                    </v-card-text>
+                    <v-card-actions>
+                      <v-spacer />
+                      <v-btn flat class="orange--text" :loading="educatorLoading" type="submit">Sign In</v-btn>
+                    </v-card-actions>
+                  </v-card>
+                </form>
               </v-tabs-content>
               <v-tabs-content id="create-an-account">
-                <v-card flat>
-                  <v-card-text>
-                    <v-text-field name="username" label="Email address" v-model.lazy="educators.username"
-                      type="email" required :rules="[() => rejected ? '' : true]"></v-text-field>
-                    <v-text-field name="password" label="Password" v-model.lazy="educators.password"
-                      type="password" required :rules="[() => rejected ? '' : true]"></v-text-field>
-                    <v-text-field name="password" label="Confirm password" type="password" required v-model.lazy="educators.confirmPassword"
-                      :rules="[validateConfirmPassword]"></v-text-field>
-                  </v-card-text>
-                </v-card>
+                <form @submit.prevent="createEducatorAccount">
+                  <v-card flat>
+                    <v-card-text>
+                      <v-text-field name="username" label="Email address (please use your school email)" v-model.lazy="educators.username"
+                        type="email" required :rules="[validateEmail]"></v-text-field>
+                      <v-text-field name="password" label="Password" v-model.lazy="educators.password"
+                        type="password" required :rules="[() => rejected ? '' : true]"></v-text-field>
+                      <v-text-field name="password" label="Confirm password" type="password" required v-model.lazy="educators.confirmPassword"
+                        :rules="[validateConfirmPassword]"></v-text-field>
+                      <p :class="displayEducatorMessage">
```

Comment:
`:class="{'green--text': unauthenticated, 'red--text': !unauthenticated}" class="auth-error"`

It may be more verbose, but avoid having styles in the js section.

Reply:
👍